gitops/generate_build.go

```diff
@@ -71,18 +75,32 @@ func GenerateBuild(fs afero.Fs, outputFolder string, component appstudiov1alpha1
 	return nil
 }
 
+// GenerateInitialBuildPipelineRun generates pipeline run for initial build of the component.
+func GenerateInitialBuildPipelineRun(component appstudiov1alpha1.Component) tektonapi.PipelineRun {
```

Comment:
If you separate this out, this won't be counted into code coverage, you'll need to add a small unit test for this :)

Reply:
A test for test... ok